Arrowgene.Ddon.Shared/Entity/Structure/CDataMDataCraftGradeUpRecipe.cs

```diff
@@ -12,11 +12,11 @@ public CDataMDataCraftGradeupRecipe()
         }
         public uint RecipeID { get; set; }
         public uint ItemID { get; set; }
-        public uint Unk0 { get; set; } // This appears to be "NextGrade", but that might be stored in the DB? 
+        public uint Upgradable { get; set; } // 1 is yes, 0 is no. will prompt a warning "will become max grade" 
```

Comment:
Can you turn this into an enum so it's easier to understand in code and debugger?